### PR TITLE
AB#50891

### PR DESCRIPTION
**Pull request by AlexandreSanscartier**
Re-added the upload a new document button

### CODE REVIEW

File: CDTS Blazor/Pages/UploadDocument.razor [hunk 1]
```diff
@@ -108,7 +108,7 @@
 
                     <div class="form-row">
 
-                        <div class="@((model.UploadedFiles != null && model.UploadedFiles.Any()? "col-md-3" : "col-md-2"))">
+                        <div class="@((model.UploadedFiles != null && model.UploadedFiles.Any()? "col-md-3" : "col-md-3"))">
```
Comment (tc-chris) [inline CDTS Blazor/Pages/UploadDocument.razor:111]: Why did your remove col-md-2?

File: CDTS Blazor/Pages/UploadDocument.razor [hunk 2]
```diff
@@ -123,7 +123,7 @@
                             <InputFile class="form-control" style="display:none;" OnChange="HandleFileSelected" id="@nameof(model.UploadedFiles)" accept=".jpg, .png, .gif, .pdf" />
                         </div>
 
-                        <div class="@((model.UploadedFiles != null && model.UploadedFiles.Any()? "col-md-9" : "col-md-10"))">
+                        <div class="@((model.UploadedFiles != null && model.UploadedFiles.Any()? "col-md-9" : "col-md-9"))">
```
Comment (tc-chris) [inline CDTS Blazor/Pages/UploadDocument.razor:126]: Something here col-md-10 to col-md-9?

File: CDTS Blazor/Pages/UploadDocument.razor (hunk 1 above)
Comment (AlexandreSanscartier) [reply CDTS Blazor/Pages/UploadDocument.razor:111]: Doesn't fit on the French Version

File: CDTS Blazor/Pages/UploadDocument.razor [hunk 3]
```diff
@@ -143,24 +143,26 @@
                         </p>
                     }
 
-                </div>
+                <button id="add_another_file" type="button" class="btn btn-light" onclick="document.getElementById('@nameof(model.UploadedFiles)').click();">@PageLocalizer["AddAnotherDocument"]</button>
```
[the remark this answers is not in the document]
Comment (tc-chris) [reply CDTS Blazor/Pages/UploadDocument.razor:146]: We need to remove that other "Add another document" button at the end. From UI design the text changes

File: CDTS Blazor/Pages/UploadDocument.razor (hunk 2 above)
Comment (AlexandreSanscartier) [reply CDTS Blazor/Pages/UploadDocument.razor:126]: See above comment, but essentially it creates a disconnect between FR and EN, but we can put it back to what it was.

File: CDTS Blazor/Pages/UploadDocument.razor (hunk 1 above)
Comment (tc-chris) [reply CDTS Blazor/Pages/UploadDocument.razor:111]: Okay, I see. We can leave it as is than

File: CDTS Blazor/Pages/UploadDocument.razor (hunk 3 above)
[the remark this answers is not in the document]
Comment (tc-tibo) [reply CDTS Blazor/Pages/UploadDocument.razor]: @tc-chris - the button is still [there per UX](https://www.figma.com/file/j5IJeZu57CtBB5fzEqRR6j/Alpha-MVP---Online-Document-Submission?node-id=0%3A1):

image

[the remark this answers is not in the document]
Comment (tc-tibo) [reply CDTS Blazor/Pages/UploadDocument.razor]: No it did not

**Review by tc-tibo — APPROVED** [review]

**Pull request merged**